### PR TITLE
[ADD] sport_club: Sport Club

### PR DESCRIPTION
**Pull request by prra-odoo**
This setup is for sports clubs that rent courts to their players, handle some basic gears and have a cafeteria.
As a small company, The Club still has quite a lot of activities to manage and does all this with Odoo. 

Task-3449717

### CODE REVIEW

File: sport_club/data/ir_attachment_pre.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="ir_attachment_498" model="ir.attachment">
+        <field name="name">unsplash_M2x3A8Q4JbY_.jpg</field>
+        <field name="datas" type="base64" file="sport_club/static/src/binary/ir_attachment/498-unsplash_M2x3A8Q4JbY_.jpg"/>
+        <field name="url">/unsplash/M2x3A8Q4JbY/.jpg</field>
```
Comment (jva-odoo) [inline sport_club/data/ir_attachment_pre.xml:6]:
```suggestion
```

File: sport_club/data/ir_ui_menu.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="website.menu_website_global_configuration" model="ir.ui.menu">
+        <field name="name">Configuration</field>
+        <field name="groups_id" eval="[(6, 0, [ref('base.group_system')])]"/>
+        <field name="parent_id" ref="website.menu_website_configuration"/>
+        <field name="sequence">5</field>
+    </record>
```
Comment (jva-odoo) [inline sport_club/data/ir_ui_menu.xml:8]: not need menu of other module

File: sport_club/data/knowledge_article.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
```
Comment (jva-odoo) [inline sport_club/data/knowledge_article.xml:2]:
```suggestion
<odoo>
```

File: sport_club/data/product_template.xml [hunk 4]
```diff
@@ -0,0 +1,224 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_template_10" model="product.template" context="{'studio': True, 'create_product_product': False}">
+        <field name="name">Club Bag </field>
+        <field name="image_1920" type="base64" file="sport_club/static/src/binary/product_template/10-image_1920"/>
+        <field name="invoice_policy">order</field>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
```
Comment (jva-odoo) [inline sport_club/data/product_template.xml:7]:
```suggestion
```

File: sport_club/data/product_template.xml [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        <field name="categ_id" ref="product.product_category_all"/>
+        <field name="uom_id" ref="uom.product_uom_unit"/>
```
Comment (jva-odoo) [inline sport_club/data/product_template.xml:9]:
```suggestion
```

File: sport_club/data/product_template.xml [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+        <field name="default_code">DISC</field>
+        <field name="website_sequence">10060</field>
+        <field name="available_in_pos" eval="True"/>
+    </record>
+    <record id="point_of_sale.product_product_tip_product_template" model="product.template" context="{'studio': True, 'create_product_product': False}">
+        <field name="name">Tips</field>
+        <field name="invoice_policy">order</field>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
+        <field name="weight">0.01</field>
```
Comment (jva-odoo) [inline sport_club/data/product_template.xml:206]: remove other module records

File: sport_club/demo/appointment_resource.xml [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```
Comment (jva-odoo) [inline sport_club/demo/appointment_resource.xml:2]:
```suggestion
<odoo noupdate="1">
```

File: sport_club/demo/appointment_type.xml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```
Comment (jva-odoo) [inline sport_club/demo/appointment_type.xml:2]:
```suggestion
<odoo  noupdate="1">
```

File: sport_club/demo/appointment_type.xml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+        <field name="location_id" ref="base.main_partner"/>
+        <field name="max_schedule_days">21</field>
+        <field name="appointment_invite_ids" eval="[(6, 0, [ref('sport_club.appointment_invite_1')])]"/>
+        <field name="resource_manage_capacity" eval="True"/>
+        <field name="name">Friday Night Free Play</field>
+        <field name="assign_method">time_auto_assign</field>
+        <field name="schedule_based_on">resources</field>
+        <field name="appointment_duration">0.5</field>
+        <field name="avatars_display">hide</field>
+        <field name="message_intro">&lt;p&gt;Meet new players!&lt;br&gt;Every Friday evening, come alone or with friends and get to play with new players.&lt;/p&gt;</field>
```
Comment (jva-odoo) [inline sport_club/demo/appointment_type.xml:24]: CDATA

File: sport_club/demo/crm_lead.xml [hunk 10]
```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="crm_lead_1" model="crm.lead">
+        <field name="name">Hello</field>
+        <field name="mobile">lna</field>
+        <field name="email_from">"Luc Nailis" &lt;lna@odoo.com&gt;</field>
+        <field name="team_id" ref="sales_team.team_sales_department"/>
+        <field name="iap_enrich_done" eval="True"/>
```
Comment (jva-odoo) [inline sport_club/demo/crm_lead.xml:8]:
```suggestion
```

File: sport_club/demo/res_partner.xml [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_partner_7" model="res.partner">
+        <field name="name">John Doe</field>
+        <field name="category_id" eval="[(6, 0, [])]"/>
+        <field name="calendar_last_notif_ack">2023-09-20 13:12:34</field>
```
Comment (jva-odoo) [inline sport_club/demo/res_partner.xml:6]:
```suggestion
```

File: sport_club/demo/res_partner.xml [hunk 12]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_partner_7" model="res.partner">
+        <field name="name">John Doe</field>
+        <field name="category_id" eval="[(6, 0, [])]"/>
```
Comment (jva-odoo) [inline sport_club/demo/res_partner.xml:5]:
```suggestion
```

File: sport_club/demo/res_partner.xml [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="calendar_last_notif_ack">2023-09-20 13:12:34</field>
+        <field name="property_stock_customer" ref="stock.stock_location_customers"/>
+        <field name="property_stock_supplier" ref="stock.stock_location_suppliers"/>
+    </record>
+    <record id="res_partner_8" model="res.partner">
+        <field name="name">lna</field>
+        <field name="category_id" eval="[(6, 0, [])]"/>
+        <field name="calendar_last_notif_ack">2023-10-12 09:19:28</field>
+        <field name="email">lna@odoo.com</field>
+        <field name="mobile">lna</field>
```
Comment (jva-odoo) [inline sport_club/demo/res_partner.xml:15]:
```suggestion
```

File: sport_club/demo/sale_order_line.xml [hunk 14]
```diff
@@ -0,0 +1,42 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_1" model="sale.order.line" context="{'studio': True, 'skip_task_generation': True, 'skip_purchase_generation': True}">
+        <field name="name">Racket "Furiosa" Option: Pack of 3 balls</field>
+        <field name="product_id" ref="sport_club.product_product_2"/>
+        <!-- <field name="state">sale</field> -->
```
Comment (jva-odoo) [inline sport_club/demo/sale_order_line.xml:6]:
```suggestion
```

File: sport_club/demo/website.xml [hunk 15]
```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="website.default_website" model="website">
+        <field name="name">industry-padel-club</field>
+        <field name="salesteam_id" ref="sales_team.salesteam_website_sales"/>
+        <field name="crm_default_team_id" ref="sales_team.team_sales_department"/>
```
Comment (jva-odoo) [inline sport_club/demo/website.xml:6]:
```suggestion
```

File: sport_club/demo/website_ir_attachments.xml [hunk 16]
```diff
@@ -0,0 +1,627 @@
...
+<odoo>
+    <record id="ir_attachment_395" model="ir.attachment">
+        <field name="name">theme_common.image_content_02</field>
+        <field name="type">url</field>
+        <field name="url">/theme_real_estate/static/src/img/content/content_img_02.jpg</field>
+        <field name="res_model">ir.ui.view</field>
+        <field name="key">theme_common.image_content_02</field>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="public" eval="True"/>
+    </record>
```
Comment (jva-odoo) [inline sport_club/demo/website_ir_attachments.xml:11]: remove all record with theme url

File: sport_club/demo/website_theme_apply.xml [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+            ('url', '=', '/shop'),
+        ]).id"/>
+        <value eval="{'name': 'Gear'}"/>
+    </function>
+</odoo>
+<!-- #fab803 yellow 1
+#f5f4f0 3
+#1a1423 2
+#FFFFFF 4
+#2e2714 5
```
Comment (jva-odoo) [inline sport_club/demo/website_theme_apply.xml:30]:
```suggestion
```

File: sport_club/demo/website_view.xml [hunk 18]
```diff
@@ -0,0 +1,379 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
```
Comment (jva-odoo) [inline sport_club/demo/website_view.xml:2]:
```suggestion
<odoo>
```

**Review by jva-odoo — CHANGES_REQUESTED** [review]

File: sport_club/data/ir_attachment_pre.xml [hunk 19]
```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```
Comment (jva-odoo) [inline sport_club/data/ir_attachment_pre.xml:2]:
```suggestion
<odoo noupdate="1">
```

**Review by jva-odoo — CHANGES_REQUESTED** [review]

File: sport_club/demo/sale_order_line.xml [hunk 20]
```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_1" model="sale.order.line" context="{'studio': True, 'skip_task_generation': True, 'skip_purchase_generation': True}">
```
Comment (jva-odoo) [inline sport_club/demo/sale_order_line.xml:3]:
```suggestion
    <record id="sale_order_line_1" model="sale.order.line">
```

File: sport_club/data/product_template.xml [hunk 21]
```diff
@@ -0,0 +1,152 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_template_10" model="product.template" context="{'studio': True, 'create_product_product': False}">
```
Comment (jva-odoo) [inline sport_club/data/product_template.xml:3]:
```suggestion
    <record id="product_template_10" model="product.template" context="{'create_product_product': False}">
```

File: sport_club/data/knowledge_article.xml [hunk 22]
```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="knowledge_article_1" model="knowledge.article" context="{'skip_check_member_access': True}">
```
Comment (jva-odoo) [inline sport_club/data/knowledge_article.xml:3]:
```suggestion
    <record id="knowledge_article_1" model="knowledge.article">
```

**Review by jva-odoo — CHANGES_REQUESTED** [review]

**Review by jva-odoo — APPROVED** [review]

**Comment by jco-odoo** [comment]
sports_club?

**Comment by xavieralt** [comment]
> sports_club?

I'll rename it :ok_hand:

**Pull request merged**